src/codegen/llvm/codegen_llvm_visitor.hpp

```diff
@@ -45,10 +49,20 @@ class CodegenLLVMVisitor: public visitor::ConstAstVisitor {
     // Output directory for code generation
     std::string output_dir;
 
-    // result string for demo
-    std::string result_code;
+private:
+    std::unique_ptr<llvm::LLVMContext> context = std::make_unique<llvm::LLVMContext>();
+
+    std::unique_ptr<llvm::Module> module = std::make_unique<llvm::Module>(mod_filename, *context);
+
+    llvm::IRBuilder<> builder;
+
+    // Stack to hold visited values
+    std::vector<llvm::Value*> values;
+
+    // Mappings for named values for lookups
+    std::map<std::string, llvm::Value*> namedValues;
```

Comment:
`namedValues` -> `named_values` i.e. camel case for variable names
Just fyi : we use [this coding conventions](https://github.com/BlueBrain/hpc-coding-conventions/blob/master/cpp/NamingConventions.md).

Reply:
Thanks! Forgot to change that one